doc/cabal-project.rst

```diff
@@ -218,8 +218,8 @@ version control.
         tag: e76fdc753e660dfa615af6c8b6a2ad9ddf6afe70
         post-checkout-command: autoreconf -i
 
-cabal-install 3.4 sdists the ``source-repository-package`` repositories and uses resulting tarballs as project packages.
-This allows sharing of packages across different projects.
+cabal-install 3.4 sdists the ``source-repository-package`` repositories provided and uses the resulting tarballs as project packages.
```

Comment:
I'm not a fan of

> cabal-install 3.4 sdists the ``source-repository-package`` repositories...

for several reasons.

First, is it only 3.4 or some other versions? I'd say:

> Since version 3.4, cabal-install sdists the ``source-repository-package`` repositories...

TBH it could do without a version altogether, perhaps. But needs checking that this version information is actually reflected somewhere. So, I'm threading safe here and leaving the version information in.

Second, I don't like derivation of words from commands: this makes text harder to read imo. How about:

> Since version 3.4, cabal-install creates tarballs for every package coming from a ``source-repository-package`` stanza (effectively applying ``cabal sdists`` to such packages).

Reply:
```suggestion
Since version 3.4, cabal-install creates tarballs for each package coming from a source-repository-package stanza (effectively applying cabal sdists to such packages).
```